src/library/Uno.Themes.WinUI.Markup/Theme/DatePicker.cs

```diff
@@ -12,188 +13,96 @@ public static class DatePicker
-					public static ThemeResourceKey<double> MinWidth => new("DatePickerFlyoutPresenterMinWidth");
-
-					[ResourceKeyDefinition(typeof(double), "DatePickerFlyoutPresenterMaxHeight")]
-					public static ThemeResourceKey<double> MaxHeight => new("DatePickerFlyoutPresenterMaxHeight");
-
-					[ResourceKeyDefinition(typeof(double), "DatePickerFlyoutPresenterAcceptDismissHostGridHeight")]
-					public static ThemeResourceKey<double> AcceptDismissHostGridHeight => new("DatePickerFlyoutPresenterAcceptDismissHostGridHeight");
-
-					[ResourceKeyDefinition(typeof(double), "DatePickerFlyoutPresenterHighlightHeight")]
-					public static ThemeResourceKey<double> HighlightHeight => new("DatePickerFlyoutPresenterHighlightHeight");
-
-
-				}
-
 				public static class Button
 				{
 					public static class Background
 					{
-						[ResourceKeyDefinition(typeof(Brush), "DatePickerButtonBackground")]
-						public static ThemeResourceKey<Brush> Default => new("DatePickerButtonBackground");
+						[ResourceKeyDefinition(typeof(Brush), "DatePickerButtonBackgroundDisabled")]
+						public static ThemeResourceKey<Brush> Disabled => new("DatePickerButtonBackgroundDisabled");
 
 						[ResourceKeyDefinition(typeof(Brush), "DatePickerButtonBackgroundPointerOver")]
 						public static ThemeResourceKey<Brush> PointerOver => new("DatePickerButtonBackgroundPointerOver");
 
 						[ResourceKeyDefinition(typeof(Brush), "DatePickerButtonBackgroundPressed")]
 						public static ThemeResourceKey<Brush> Pressed => new("DatePickerButtonBackgroundPressed");
-
-						[ResourceKeyDefinition(typeof(Brush), "DatePickerButtonBackgroundDisabled")]
-						public static ThemeResourceKey<Brush> Disabled => new("DatePickerButtonBackgroundDisabled");
 					}
-
 					public static class Foreground
 					{
-						[ResourceKeyDefinition(typeof(Brush), "DatePickerButtonForeground")]
-						public static ThemeResourceKey<Brush> Default => new("DatePickerButtonForeground");
+						[ResourceKeyDefinition(typeof(Brush), "DatePickerButtonForegroundDisabled")]
+						public static ThemeResourceKey<Brush> Disabled => new("DatePickerButtonForegroundDisabled");
 
 						[ResourceKeyDefinition(typeof(Brush), "DatePickerButtonForegroundPointerOver")]
 						public static ThemeResourceKey<Brush> PointerOver => new("DatePickerButtonForegroundPointerOver");
 
 						[ResourceKeyDefinition(typeof(Brush), "DatePickerButtonForegroundPressed")]
 						public static ThemeResourceKey<Brush> Pressed => new("DatePickerButtonForegroundPressed");
-
-						[ResourceKeyDefinition(typeof(Brush), "DatePickerButtonForegroundDisabled")]
-						public static ThemeResourceKey<Brush> Disabled => new("DatePickerButtonForegroundDisabled");
 					}
+				}
+				public static class ButtonBorderBrush
+				{
+					[ResourceKeyDefinition(typeof(Brush), "DatePickerButtonBorderBrush")]
+					public static ThemeResourceKey<Brush> Default => new("DatePickerButtonBorderBrush");
```

Comment:
Are we going to be able to generate the code to "hide" the .Default behind those nested classes + implicit operators like we talked about with @dansiegel?